contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -120,47 +70,43 @@ def _clone_repo(repo_url, verifyssl=True, branch='master'):
         Repo.clone_from(repo_url, abs_local_path, branch=branch, depth=1)
         return abs_local_path
 
-    def _move_packs(self, abs_repo_base, packs, abs_local_path, subtree):
+    def _move_pack(self, abs_repo_base, pack_name, abs_local_path):
```

Comment:
The logic remains "clone to temp, move to `opt/stackstorm/packs`? If so, why? why not clone to the location? Again, the requirement is I go to `/opt/stackstorm/packs/mypack` and do `git status`there, and all the other git operations. It may still  work as we move `.git`, just want to better understand what is a good reason of intermediate copy?